README.md

```diff
@@ -47,49 +47,50 @@ We are _following_ a **_battle-tested_ industry-standard** approach
- [Matthew Green](https://blog.cryptographyengineering.com/),
- [Niels Ferguson](https://en.wikipedia.org/wiki/Niels_Ferguson)
- and [Bruce Schneier](https://www.schneier.com/blog/about/)
-  + "Under the hood" we are using Erlang's
-[crypto](http://erlang.org/doc/man/crypto.html) library
-_specifically_ AES with **256 bit keys** <br />
-(_the same as AWS or Google's KMS service_)
-see: http://erlang.org/doc/man/crypto.html#block_encrypt-4
-+ Password "hashing" using the **Argon2**
-key derivation function (KDF): https://en.wikipedia.org/wiki/Argon2 <br />
-_specifically_ the Elixir implementation of `argon2`
-written by David Whitlock: https://github.com/riverrun/argon2_elixir
-which in turn uses the **C** "_reference implementation_"
-as a "Git Submodule".
+* **Advanced Encryption Standard** (**AES**) to encrypt sensitive data.
+  * **Galois/Counter Mode**
+    for _symmetric_ key cryptographic block ciphers:
+    https://en.wikipedia.org/wiki/Galois/Counter_Mode
+    recommended by many security and cryptography practitioners including
+    [Matthew Green](https://blog.cryptographyengineering.com/),
+    [Niels Ferguson](https://en.wikipedia.org/wiki/Niels_Ferguson)
+    and [Bruce Schneier](https://www.schneier.com/blog/about/)
+  * "Under the hood" we are using Erlang's
+    [crypto](http://erlang.org/doc/man/crypto.html) library
+    _specifically_ AES with **256 bit keys** <br />
+    (_the same as AWS or Google's KMS service_)
+    see: http://erlang.org/doc/man/crypto.html#block_encrypt-4
+* Password "hashing" using the **Argon2**
+  key derivation function (KDF): https://en.wikipedia.org/wiki/Argon2 <br />
+  _specifically_ the Elixir implementation of `argon2`
+  written by David Whitlock: https://github.com/riverrun/argon2_elixir
+  which in turn uses the **C** "_reference implementation_"
+  as a "Git Submodule".
 
 > `¯\_(ツ)_/¯...?` Don't be "put off" if any of these terms/algorithms
-are _unfamiliar_ to you; <br />
-this example is "***step-by-step***"
-and we are _happy_ to answer/clarify
-_any_ (_relevant and specific_) questions you have!
+> are _unfamiliar_ to you; <br />
+> this example is "**_step-by-step_**"
+> and we are _happy_ to answer/clarify
+> _any_ (_relevant and specific_) questions you have!
 
 #### OWASP Cryptographic Rules?
 
 This example/tutorial follows
 the Open Web Application Security Project (**OWASP**)
 Cryptographic and Password rules:
-+ [x] Use "***strong approved Authenticated Encryption***"
-based on an ***AES algorithm***.
-  + [x] Use GCM mode of operation for symmetric key cryptographic block ciphers.
-  + [x] Keys used for encryption must be rotated at least annually.
-+ [x] Only use approved public algorithm **SHA-256** or better for hashing.
-+ [x] **Argon2** is the winner of the password hashing competition
-and should be your ***first choice*** for **_new_ applications**.
+
+* [x] Use "**_strong approved Authenticated Encryption_**"
```

Comment:
@Cleop which Markdown formatter are you using that automatically changes `+` (_bullet points_) to `*`...?

Reply:
Nice points re plugins, I agree. I just took a look at my packages and I have a grand total of 5 linters installed! My guess would be that this is from 'prettier-atom' which is the one I remember adding most recently for one of our projects. It changes a lot more than any other linters I've used before I think. I think picking some as a team would be great 👍

Reply:
please open an issue on Linters in: https://github.com/dwyl/technology-stack/issues
so we can _capture_ the discussion centrally. thanks.